tests/istanbul/eip152_blake2/test_blake2_delegatecall.py

```diff
@@ -0,0 +1,57 @@
+"""abstract: Test delegatecall to Blake2B Precompile before and after it was added."""
```

Comment:
This test covers https://github.com/ethereum/tests/blob/develop/src/GeneralStateTestsFiller/stPreCompiledContracts/delegatecall09UndefinedFiller.yml